packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -76,9 +81,10 @@ const HomePage = () => {
       </ControlsWrapper>
       <ListEmpty
         // TODO
-        onClick={() => {}}
+        onClick={handleClickToggleModal}
       />
       {/* <List data={data} /> */}
```

Comment:
Comment

Reply:
It can be kept for the moment, I will change it very very soon to display the List with Card compos!